### ISSUE TITLE
Wear OS

### ISSUE THREAD

**Issue by saltsoftdrink** [twofas/2fas-android#121]
@rafakob Create this app for Wear OS also

**Comment by muesli79** [comment]
That would be great!

**Comment by JRuedas** [comment]
It would be super useful to be able to access the tokens from the Watch.

**Comment by MicaelJarniac** [comment]
That would be awesome! There's another FOSS authenticator for Android, that does have WearOS support:
https://github.com/jamie-mh/AuthenticatorPro

**Comment by adrianh-za** [comment]
Indeed, a Wear OS app or even a companion Wear OS app will be a fantastic addition to 2FAS.

**Comment by avipars** [comment]
@MicaelJarniac  but it lacks a backend, browser extension, cloud support
